Content.Server/Light/Components/ToggleableOccluderComponent.cs

```diff
@@ -0,0 +1,22 @@
+using Content.Shared.DeviceLinking;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Server.Light.Components
+{
+    /// <summary>
+    ///     Allows entities with OccluderComponent to toggle that component on and off.
+    /// </summary>
+    [RegisterComponent]
+    public sealed partial class ToggleableOccluderComponent : Component
+    {
+        [DataField]
+        public ProtoId<SinkPortPrototype> OnPort = "On";
+
+        [DataField("offPort", customTypeSerializer: typeof(PrototypeIdSerializer<SinkPortPrototype>))]
+        public string OffPort = "Off";
```

Comment:
```suggestion
        [DataField]
        public ProtoId<SinkPortPrototype>OffPort = "Off";
```
same with the other datafiled below
also add documentation to the datafields